client/src/components/Login/SignUpForm/index.js

```diff
@@ -59,11 +61,60 @@ const SignUpForm = ({ switchView }) => {
     }
   };
 
```

Comment:
You will also need to copy over the useEffect from loginForm. It should look something like this. (This code will enable the submit button if all fields have been properly filled out.)

  useEffect(() => {
    setIsDisabled(
      validationState.password.error || validationState.username.error  || validationState.email.error
    );
  }, [validationState]);